templates/adspot/handlers/manage.ts

```
@@ -0,0 +1,79 @@
+    if (body.buttonIndex === 1) {
+        return {
+            buttons: [{ label: 'Back', action: 'post' }],
+            component: createAllBidsComponent(storage, config),
+            handler: 'manage',
+        }
+    } else if (body.buttonIndex === 2 && config.mode === 'Auction') {
+        return handleAuctionEnd(storage, config)
+    } else if (body.buttonIndex === 2) {
+        return handleBidAcceptance(storage, config, body.inputText)
+    }
+    throw new Error('Invalid action for owner')
```

Biome

[error] This else clause can be omitted because previous branches break early.

Unsafe fix: Omit the else clause.


(lint/style/noUselessElse)